packer/media-service/media-service-ami.sh

```diff
@@ -8,10 +8,6 @@ function new_section {
 
 set -e
 
-new_section "Moving config files into place"
-mv /tmp/media-service_logrotate /etc/logrotate.d/media-service
-sudo chown root:root /etc/logrotate.d/media-service
```

Comment:
if it goes to syslog now, are we using the syslog rotator or logrotate (per your comment below)?
I know very little about this (i.e. the blog posts i just read), so maybe we just talk about it IRL